### ISSUE TITLE
Are point normals and class-label one-hot vectors used for ShapeNet Parts segmentation task?

### ISSUE THREAD

**Issue by daerduoCarey** [yangyanli/PointCNN#93]
After I read the code, I think for the ShapeNet Part segmentation task, you don't even use the point normals and class-label one-hot vectors as inputs. Is this correct? I think PointNet++ and many other methods use them as inputs as well for the network training. Do you think such additional information would help PointCNN to obtain even better results?

Thank you!

**Comment by yangyanli** [comment]
Hi Kaichun (@daerduoCarey),

We tried using normals before, and found it improves the performance only a little bit. We suspect that the normals are something that can be derived from the points--kind of duplicate information, thus not that useful.

As for the class label, we also tried it at some point during the design of PointCNN, and we found it does not improve the overall performance. However, this conclusion might be wrong with the latest PointCNN, which we didn't try.

Anyway, we found the datasets are not as large as we want to demonstrate the effectiveness of different designs, with some of them are getting quite saturated, especially ModelNet40. It would be great if more comprehensive datasets could be released for benchmarking.
